### ISSUE TITLE
Given there are two wrecks side by side, when one is done and user finishes scavenging the 2nd on,e 1st wreck can be scavenged again

### ISSUE THREAD

**Issue by yukihito23** [salutesh/Exile_Scavenge#3]
Description:
Given scenario where there are two wrecks side by side.
When user scavenged 1st wreck, then goes and scavenged 2nd wreck, the 1st wreck can be scavenged again, which opens potential of as long as there are two wrecks existing in similar location, user can sit there and keep scavenging.
Env:
Ver: 2017/10/16
Env: Exile 1.0.3 / Chernarus Redux v0.41

Steps to Repro:
1. On server, find location where there are two wrecks side by side
2. Scavenged wreck 1
3. scavenge wreck 2
4. observe wreck 1 again

expected results:
the object ID check which blocks user from scavenging again on something they've looked through before, seems to reset, thus which allows user to scavenge object they have already looked at again.

actual results:
object shall not allow user to scavenge anymore even if they go scavenge another object in between.

**Comment by salutesh** [comment]
There is a cleanup for the variable that stores the already scavenged objects for each player inside the ScavangeActionMonitor. I will remove it because it does not fit the system.

```
if ( count (player getVariable ["ScavangedObjects", []]) > 10 ) then {
	player setVariable ["ScavangedObjects", []];
};
```

**Comment by yukihito23** [comment]
with latest updated version, this behavior can still be seen, rejecting.

**Comment by salutesh** [comment]
Will try again to find a workaround for this.

**Comment by salutesh** [comment]
This sould be fixed in version 0.6 now. Need confirmation!

**Comment by salutesh** [comment]
Fixed!
